omega/omega-transaction/src/main/java/org/apache/servicecomb/pack/omega/transaction/CallbackContext.java

```diff
@@ -15,13 +15,14 @@
  * limitations under the License.
  */
 
-package org.apache.servicecomb.pack.omega.context;
```

Comment:
We need to add a old class with deprecated annotation in org.apache.servicecomb.pack.omega.context to provide backward compatible support.

Reply:
> We need to add a old class with deprecated annotation in org.apache.servicecomb.pack.omega.context to provide backward compatible support.

This is a good idea otherwise I will modify a lot of tests cases（Although it has been modified a lot 😢）

I think this feature only needs to support the SAGA state machine. For this, We need to add a parameter `alpha.feature.akka.transactions = saga, tcc`

Currently, this feature is enabled with the following two parameters:

```
alpha.feature.akka.transactions=saga
alpha.feature.akka.enabled=true
```
What do you think?

Reply:
+1, we can create another JIRA to "enable the protocol separately" to track it.

Reply:
I will add the compensation confirm event only in the state machine mode in this PR, It's better for small step iterations.

Reply:
done